ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/documentation/management-security.j2

```diff
@@ -25,8 +25,73 @@
 
 | SSL Profile Name | TLS protocol accepted | Certificate filename | Key filename | Cipher List |
 | ---------------- | --------------------- | -------------------- | ------------ | ----------- |
+{%         set ssl_profiles_certs = [] %}
 {%         for ssl_profile in management_security.ssl_profiles | arista.avd.natural_sort %}
 | {{ ssl_profile.name | arista.avd.default('-') }} | {{ ssl_profile.tls_versions | arista.avd.default('-') }} | {{ ssl_profile.certificate.file | arista.avd.default('-') }} | {{ ssl_profile.certificate.key | arista.avd.default('-') }} | {{ ssl_profile.cipher_list | arista.avd.default('-') }} |
+{%             set tmp_cert = {} %}
+{%             if ssl_profile.trust_certificate is arista.avd.defined %}
+{%                 set tmp_cert = {'trust_certificate': ssl_profile.trust_certificate} %}
+{%             endif %}
+{%             if ssl_profile.chain_certificate is arista.avd.defined %}
+{%                 do tmp_cert.update({'chain_certificate': ssl_profile.chain_certificate}) %}
+{%             endif %}
+{%             if tmp_cert | length > 0 %}
+{%                 do tmp_cert.update({'name': ssl_profile.name}) %}
+{%                 do ssl_profiles_certs.append(tmp_cert) %}
+{%             endif %}
+{%         endfor %}
+{%         for ssl_profile in ssl_profiles_certs | arista.avd.natural_sort('name') %}
+
+### SSL profile {{ ssl_profile.name }} Certificates Summary
+
+{%             if ssl_profile.trust_certificate is arista.avd.defined %}
```

Comment:
I propose to move the newline after the if to avoid 2 newlines if no trust_ceritificates are present.
```suggestion
{%             if ssl_profile.trust_certificate is arista.avd.defined %}

```